config/config.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2016 Mender Software AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package config
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+	"time"
+)
+
+var (
+	Config = viper.New()
```

Comment:
Yes and yes. 

I'd rather use vanilla viper TBH, I might do this in `useradm` actually.

I'd factor out `config` as is however, so that exisiting services can be migrated to go-lib-micro with minimal effort.